classes/pref/mobile.php

```diff
@@ -1,8 +1,10 @@
 <?php
 class Pref_Mobile extends Handler_Protected {
-    function csrf_ignore($method) {
-        return $method === "index";
-    }
+    function csrf_ignore(string $method): bool {
+		$csrf_ignored = array("index");
+
+		return array_search($method, $csrf_ignored) !== false;
+	}
```

Comment:
Indentation.

Reply:
This was a spaces vs. tabs crime, it looked fine on my machine.

Reply:
I figured it was something like that